src/SFA.DAS.ApprenticeCommitments.Types/SFA.DAS.ApprenticeCommitments.Types.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What's the rationale behind the new Types project?

If it's to share with the Web layer I wonder why we would do that now when we haven't shared any of the DTO types before?

I can see that there is a case to be made for sharing the types (and also some arguments against doing so) - however I imagine that it would have to be *all* of the types and not just this one.  I would think that sharing *one* of the types and not the rest might be the worst of both worlds, wouldn't you?

Reply:
The only rational for creating a Types project is because of the ChangeOfCircumstanceNotifications enum, it would be really easy to get these states wrong in the consuming layer (wrong order etc). And also it's being set in the domain layer so the logic is buried here.

Reply:
I don't really see a difference between an enum and the rest of the API surface to be honest.  It would be just as easy to have a typo in a property name, or mis-type a datetime as string or similar.

In my opinion, coupling the Web layer to the API layer should be carefully considered as there are cons as well as pros.  I also think that it should be a complete set of types used by the API, rather than a single type - that just creates confusion and complexity.